xbmc/playlists/SmartPlayList.cpp

```diff
@@ -726,7 +726,7 @@ std::string CSmartPlaylistRule::FormatWhereClause(const std::string &negate, con
     table = "songview";
 
     if (m_field == FieldGenre)
-      query = negate + " EXISTS (SELECT 1 FROM song_genre, genre WHERE song_genre.idSong = " + GetField(FieldId, strType) + " AND song_genre.idGenre = genre.idGenre AND genre.strGenre" + parameter + ")";
+      query = negate + " EXISTS (SELECT 1 FROM song_genre, genre WHERE song_genre.idSong = " + GetField(FieldId, strType) + " AND song_genre.idGenre = genre.idGenre AND genre.name" + parameter + ")";
```

Comment:
No, it's just not sync across music and videodb. I'd not change it right now without a real need.